gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/model/StoreTypesEndpointResponse.java

```diff
@@ -16,18 +16,31 @@
 
 package uk.gov.gchq.gaffer.gaas.model;
 
+import org.springframework.stereotype.Component;
 import java.util.List;
 
+@Component
 public class StoreTypesEndpointResponse {
 
-    private List<String> storetypes;
+    private List<String> storeTypes;
```

Comment:
I think this should be changes as described in https://github.com/gchq/gaffer-experimental/issues/108#issuecomment-858651495.